report/thesis.tex

```diff
@@ -19,12 +19,12 @@
 
 \begin{document}
 \maketitle
-% \makededication
-% \makeacks
 
 \begin{abstract}
 
-TODO
+This report presents the fork accountability problem, which concerns with the detection of nodes responsible of a fork in a distributed network of machines running BFT consensus when the system is compromised. We analyze and study such a challenge in the context of the Tendermint protocol, whose consensus algorithm does not currently have any mechanism to make all replicas accountable of their actions in the case of a fork when more than one third of machines in the system are faulty. 
+
+We are going to focus on describing the design of a fork accountability solution in Tendermint that was conceived by researchers from the École Polytechnique Fédérale de Lausanne and Informal Systems and show how this algorithm can guarantee the detection of at least f + 1 faulty processes in a completely asynchronous system. The core part of this work focuses on validating this proposal with a proof-of-concept environment where the algorithm was developed and tested along with several features. Finally, some benchmarks and experiments have been carried out in order to give the idea of the computational complexity of the solution from a performance point of view.
```

Comment:
```suggestion
We are going to focus on describing the design of a fork accountability solution for Tendermint consensus algorithm, that was conceived by researchers from the École Polytechnique Fédérale de Lausanne and Informal Systems. This solution can guarantee the detection of at least f + 1 faulty processes in a completely asynchronous system. The core contribution of this work consists on validating this proposal with a proof-of-concept implementation where the algorithm was developed and tested along with several features. Finally, some benchmarks and experiments have been carried out in order to give the idea of the computational complexity of the solution from a performance point of view.
```

Reply:
I think we should avoid "f + 1" here. The reader is not familiar with "f" parameter.

Reply:
Fixed, Jovan is right we should not mention f yet (fixed it as well).